app/controllers/payment-links/metadata/metadata-form.js

```diff
@@ -2,21 +2,39 @@ const fields = {
   metadataKey: {
     id: 'metadata-column-header',
     name: 'metadata-column-header',
-    validation: [ {
-      validator: isNotEmpty,
-      message: 'Enter a column header'
-    } ]
+    validation: [
+      {
+        validator: isNotEmpty,
+        message: 'Enter a column header'
+      },
+      {
+        validator: isValidLength,
+        message: 'Column header must be 50 characters or fewer'
```

Comment:
It would be good for this to use the value in `META_DATA_MAX_LENGTH` to avoid duplication and make updating this in the future simpler

Reply:
Good idea - done.